Makefile

```diff
@@ -4,7 +4,7 @@ VERSION=
 
 MAKEFILES_VERSION=4.8.0
 
-.DEFAULT_GOAL:=compile
+.DEFAULT_GOAL:=help
```

Comment:
I don't think it will break existing ones, because the main Makefile is not altered when you upgrade the makefiles, only the ones in `build/make`. So this will only apply to new repositories which import the makefiles for the first time.

Reply:
Ah I see! You are totally right, this is more the kind of initial makefiles usage. Then we have any change here :D